lib/decoder.js

```diff
@@ -213,6 +224,10 @@ Decoder.prototype.decode = function(len, data, callback) {
     }
   }
 
+  if(len === 11) {
```

Comment:
Very good, it was still in place when I looked at things.
Just reject my PR then, I'll work based on the UNKN data.